modules/migration-viewing-migration-crs.adoc

```diff
@@ -1,20 +1,64 @@
 // Module included in the following assemblies:
 //
-// migration/migrating-3-4/migrating-openshift-3-to-4.adoc
-// migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc
-// migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4_1-4/troubleshooting.adoc
+// migration/migrating-4_2-4/troubleshooting.adoc
 [id='migration-viewing-migration-crs_{context}']
-= Viewing migration custom resources
+= Viewing migration Custom Resources
 
-To view a migration custom resource:
+The CAM tool creates the following Custom Resources (CRs) for migration:
 
+image::migration-architecture.png[migration architecture diagram]
+
+image:darkcircle-1.png[20,20] link:https://github.com/fusor/mig-controller/blob/master/pkg/apis/migration/v1alpha1/migcluster_types.go[MigCluster] (configuration, CAM cluster): Cluster definition
```

Comment:
The GitHub links are a red flag for me, as I've been told we don't link externally to them. See https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Reply:
True, but this was approved by the PM, because the user will need to access the migration CRs for troubleshooting information. The info does not exist anywhere else and is open to change.